command/polybftmanifest/manifest_init.go

```diff
@@ -0,0 +1,211 @@
+package polybftmanifest
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/genesis"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/spf13/cobra"
+)
+
+const (
+	manifestPathFlag      = "path"
+	premineValidatorsFlag = "premine-validators"
+	validatorsFlag        = "validators"
+	validatorsPathFlag    = "validators-path"
+	validatorsPrefixFlag  = "validators-prefix"
+
+	defaultValidatorPrefixPath = "test-chain-"
+	defaultManifestPath        = "./manifest.json"
+
+	nodeIDLength       = 53
+	ecdsaAddressLength = 42
+	blsKeyLength       = 2
+)
+
+var (
+	params = &manifestInitParams{}
+)
+
+// GetCommand returns the rootchain emit command
+func GetCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "polybft-manifest",
```

Comment:
I would call it just `manifest` and specify in the docs, `--help`, etc, that it only applies to `polybft`, it will anyway in the future

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1005/commits/f432f4babd18c4081d8bd25c8ade9a73badfbc28